rapid_pro_tools/rapid_pro_client.py

```diff
@@ -667,8 +706,11 @@ def update_contact(self, urn, name=None, contact_fields=None):
         :param contact_fields: Dictionary of field key to new field value | None. If None, no keys are updated.
                                Keys present on the server contact but not in this dictionary are left unchanged.
         :type contact_fields: (dict of str -> str) | None
+        :param groups: list of group objects or UUIDs
```

Comment:
Can you clarify here the write behaviour - does Rapid Pro overwrite or merge?

Reply:
This will overwrite the groups in rapid_pro. If you intend to add fetch the contact and append it to the existing group. Have updated the docs. Thanks